libraries/AP_HAL_SITL/SITL_State.h

```diff
@@ -129,7 +130,7 @@ class HALSITL::SITL_State {
     uint32_t CalculateBlockCRC32(uint32_t length, uint8_t *buffer, uint32_t crc);
 
     void _update_gps(double latitude, double longitude, float altitude,
-                     double speedN, double speedE, double speedD, bool have_lock);
+                     double speedN, double speedE, double speedD, bool gps_enable);
```

Comment:
I'm not going to argue whether this is correct or not, but this does change the meaning of where it is used **for some drivers** - previously it just meant that SITL didn't had lock yet, but now it won't have GPS at all.

Reply:
On SITL, disable is no present for all other object, it should be the same. The GPS_FIX will still allow to prevent the lock